spec/forms/gpo_verify_form_spec.rb

```diff
@@ -151,12 +151,7 @@
 
       context 'ThreatMetrix rejection' do
         let(:pending_profile) do
-          create(
-            :profile,
-            user: user,
-            gpo_verification_pending_at: 1.day.ago,
-            fraud_review_pending_at: 1.day.ago,
-          )
+          create(:profile, :verify_by_mail_pending, :fraud_review_pending, user: user)
```

Comment:
Does this spec edit cover the change?

Reply:
This covers the change but I did totally forget to add a test for the new profile method. I'll put one together for that.

Reply:
And for the how it covers the change: The previous code was actually testing an impossible situation. The users who have the fraud pending timestamps will have a `fraud_pending_reason` value. They do not in this test example. As a result this misconfigured test masked the bug.

Reply:
Ah! Thanks for explaining. Does this mean that pending profiles in specs should no longer have a fraud_review_pending_at? There are more of those in gpo_verify_controller_spec and probably others.

Reply:
Users in the specs should have a `fraud_review_pending_at` for now. We have not stopped writing to that column _yet_. Eventually we will and that will be a much bigger change. We need to move the reads to `fraud_review_pending` first to enable that transition.

Reply:
The trait I used in the factory sets the `fraud_review_pending_at` column and `fraud_pending_reason` column.

Reply:
I meant, should we replace the explicit column name with the trait in specs so that it will catch impossible situations. Not in this PR though.

Reply:
Ah yes, there are others for sure. @kbighorse chased most of them down but a few slipped through for sure. We should be trying to clean them up as we find them. I'll go after the other ones in this form spec.

Reply:
I created a ticket re: the impossible situation issue: https://cm-jira.usa.gov/browse/LG-10412